content/en/blog/2020/mosn-proxy/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Introducing MOSN: An Alternative Data Plane"
```

Comment:
Makes it sound like Istio is introducing a new thing called MOSN. MOSN exists already and what's new is Istio integration. Maybe "Using MOSN with Istio: an alternative data plane".

Reply:
> Makes it sound like Istio is introducing a new thing called MOSN. MOSN exists already and what's new is Istio integration. Maybe "Using MOSN with Istio: an alternative data plane".

Done.